### PR TITLE
Vs user guide

### PR DESCRIPTION
**Pull request by phlptp**
Merge in components of v3UserGuide from separate branch.

### CODE REVIEW

File: docs/apps/Broker.md [hunk 1]
```diff
@@ -18,13 +18,13 @@ command line only:
 
 configuration:
   -n [ --name ] arg      name of the broker
-  -t [ --coretype ] arg      type of the broker ("(zmq)", "ipc", "test", "mpi","test", "tcp", "udp")
+  -t [ --type ] arg      type of the broker ("(zmq)", "ipc", "test", "mpi","test", "tcp", "udp")
```
Comment (trevorhardy) [inline docs/apps/Broker.md:21]: It looks like the command line arguments in the documentation for the helics apps have been reverted to use the (as of v3) old names.

File: docs/apps/cmdArgs.md [hunk 2]
```diff
@@ -10,13 +10,13 @@ command line only:
 
 configuration:
   -n [ --name ] arg      name of the broker
-  -t [ --coretype ] arg      type of the broker ("(zmq)", "ipc", "test", "mpi",
```
Comment (trevorhardy) [inline docs/apps/cmdArgs.md:13]: It looks like the command line arguments in the documentation have been reverted to use the (as of v3) old names.

File: docs/user-guide/broker_hierarchies.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ For examples, the config JSON for the Distribution System A (where Broker A is a
 ```JSON
 {
     "name" : "DistributionSystemA",
-    "coreinitstring": "--broker_address=tcp://127.0.0.1"
```
Comment (trevorhardy) [inline docs/user-guide/broker_hierarchies.md:30]: It looks like these sample JSONs in the  documentation for the have been reverted to use the (as of v3) old names.

File: docs/apps/Broker.md (hunk 1 above)
Comment (phlptp) [reply docs/apps/Broker.md:21]: did you make those changes in the change name branch or the v3userguide branch?

File: docs/user-guide/broker_hierarchies.md (hunk 3 above)
Comment (phlptp) [reply docs/user-guide/broker_hierarchies.md:30]: there are a lot of files that got moved around, which is why it wasn't building so I am trying to reconstruct things.  Looks like I need to do a bit more work.

File: docs/apps/Broker.md (hunk 1 above)
Comment (trevorhardy) [reply docs/apps/Broker.md:21]: I didn't intend to make those changes outside the trim_config_options branch. The line that is being changed, though, regardless of how it got there, is the (new) correct name and the edit is reverting it to the old name.

Comment (phlptp) [reply docs/apps/Broker.md:21]: Basically the files got moved so the merge didn't capture the updates yet.  I will work on that.

File: docs/user-guide/broker_hierarchies.md (hunk 3 above)
Comment (trevorhardy) [reply docs/user-guide/broker_hierarchies.md:30]: I know @allisonmcampbell has been restructuring the files associated with the User Guide; would you have any problem with her doing a review as well to make sure her changes aren't getting trampled on?

File: docs/apps/Broker.md (hunk 1 above)
Comment (trevorhardy) [reply docs/apps/Broker.md:21]: Maybe the change was made to helics3 when merging in the trim_config_options and then v3userguide was updated with those changes?

**Comment by phlptp** [comment]
@allisonmcampbell the v3userguide branch had some merges in that were interfering quite a bit so I started over based on the current helics3.  I need to make sure I am getting everything.

**Comment by trevorhardy** [comment]
Even if we ignore/ the work @allisonmcampbell  has been doing for this past week in restructuring the User Guide docs in the v3userguide branch for the purposes of getting a preliminary v3 out the door, we obviously want to preserve her work so that it can be painlessly reapplied to the appropriate branch(es).

**Comment by phlptp** [comment]
I am not touching the v3Userguide branch.  So at some point we will have to do some manual work to mimic the changes since I don't think that branch can be merged easily or at all at this point.

**Comment by trevorhardy** [comment]
Got it. We can deal with that hassle when the time comes in coming weeks.

**Comment by phlptp** [comment]
@trevorhardy  I tried to make sure the recent changes got in.  You will need to double check see if I missed any

**Review by allisonmcampbell — APPROVED** [review]

**Pull request merged**